<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditProfile</class>
 <widget class="QDialog" name="EditProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Profile</string>
  </property>
  <property name="windowIcon">
   <iconset resource="files.qrc">
    <normaloff>:/icon/images/001_45.ico</normaloff>:/icon/images/001_45.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QDialogButtonBox" name="btn_box">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>540</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>721</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_email">
          <property name="text">
           <string>Email : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="led_email"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_lastname">
          <property name="text">
           <string>Lastname : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="led_lastname"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="1">
         <widget class="QLineEdit" name="led_postcode"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_postcode">
          <property name="text">
           <string>Post code : </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_mobilenumber">
          <property name="text">
           <string>Mobile number : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="led_mobilenumber"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout_9">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_money">
          <property name="text">
           <string>Money : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spb_money">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <double>100000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="btn_increasemoney">
       <property name="text">
        <string>Increase Money</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout_7">
        <item row="0" column="1">
         <widget class="QLineEdit" name="led_address"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_address">
          <property name="text">
           <string>Address : </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout_8">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_company">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Company name : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="led_companyname"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_name">
          <property name="text">
           <string>Name : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="led_name"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_username">
          <property name="text">
           <string>Username : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="led_username">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_10">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_city">
      <property name="text">
       <string>City : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="led_city"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_11">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_state">
      <property name="text">
       <string>State : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cmb_state">
      <item>
       <property name="text">
        <string>Tehran</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ardabil</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Azerbaijan, East</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Azerbaijan, West</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bushehr</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chahar Mahaal and Bakhtiari</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fars</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gilan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Golestan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hamadan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hormozgan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ilam</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Isfahan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kerman</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kermanshah</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Khorasan, North</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Khorasan, Razavi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Khorasan, South</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Khuzestan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kohgiluyeh and Boyer-Ahmad</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kurdistan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lorestan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Markazi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mazandaran</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Qazvin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Qom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semnan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sistan and Baluchestan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alborz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yazd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zanjan</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_changepass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
